source/CommonJobs/CommonJobs.Application.EvalForm/Common.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace CommonJobs.Application.Evaluations
+{
+    internal static class Common
+    {
+        //public const string DefaultEvaluationId = "Evaluations/Default";
+
+        public const string DefaultTemplateId = "Template/Default";
+
+        public static string GenerateEvaluationId(string period, string userName)
+        {
+            return "Evaluations/" + period + "/" + userName;
+        }
+
+        public static string GenerateCalificationId(string period, string userName, string evaluator)
+        {
+            return "Evaluations/" + period + "/" + userName + "/" + evaluator;
+        }
```

Comment:
I understand now... (previous comment).

I just based this class on MyMenu. But makes sense what you're proposing. Will do it. @andresmoschini do you want to add something here?